hub/powertoys/grouppolicy.md

```diff
@@ -53,18 +110,143 @@ For each utility shipped with PowerToys, there's a "Configure enabled state" pol
 
 This policy has a higher priority than the policy "Configure global utility enabled state" and overrides it.
 
+> [!NOTE]
+> PDF file preview: There have been reports of incompatibility between the PDF Preview Handler and Outlook.
+
+#### Table of utility Policies
```

Comment:
At this point, i feel like maybe we auto generate this file based on the GP file maybe.  I'm worried about this getting out of sync

Reply:
I don't think that you can autogenerate the whole docs file as there are to complex things (Registry information, Intune information, ...) which are not part of the admx file.

Reply:
Please have in mind that the value of the second column comes technically from the ADML file and not from the ADMX file. And that the value of the first column would be the first part of the second column. (Not sure if this isn't to complex for autogeneration. 🤔)

I don't think that you can autogenerate the whole docs file as there are to complex things (Registry information, Intune information, ...) which are not part of the admx file.